no-computer2.md

```diff
@@ -0,0 +1,81 @@
+experience like now? What would their experience be like if you could help 
+them?_
+The need is to have a uniform way of writing laws. Without a system in place a persons experience would be scattered and difficult to understand and complex. The experience with a regulated system is much better.
+
+
+### Why a language?
+_Why is a DSL appropriate for your user(s)? How does it address the need?_
+A DSL is appropriate because we need to write laws in language but without guidelines the above problems would run rampant, with regulation these problems are mitigated.
+
+
+### Why you?
+_What excites you about this idea? How did you come up with it?_
+This idea is cool because its necessary, but I actually think that this system is pretty archiac and difficult for the layman to understand. I came up with this idea when I was reading some legislation for my politics class.
+
+
+### Domain
+_Describe the project's domain in five words._
+Old white dudes writing laws.
+
+
+### Interface (syntax)
+_How might the user interact with the language? What does programming look 
+like? Why is this the right way to interact with the problem domain?_ 
+The user interacts with the language by under a series of declaratives, some of which are required and unique, some of which are required* and some of which are optional. A program looks like a series of text with bolded headers starting with the sponsors of a bill, the name, the enacting clause, the substance (in a series of WHEREAS subheaders), the repealing clause (which repeals all laws conflicting), and the effective date. 
+
+
+### Operation (semantics)
+_What might happen when a program runs? How does a program interact with the
+user? What kinds of errors might occur, and how might they be communicated to
+the user?_
+When a program runs the content of the program becomes the law of the land, enforceable by police. Errors can occur in spelling, or content (bad laws) and they are communicated to the user by the executive and judicial branches or by the people.
+
+
+### Expressiveness
+_What should be easy to do in this language? What should be possible, but
+difficult? What should be impossible or very difficult?_
+It _should_ be easy to write laws but its actually not very easy because of the archaic syntax. It would be possible to write non-laws in this format but difficult, and its impossible to do pretty much anything else.
+
+
+### Related work
+_Are there any other DSLs in this domain? If not, describe how you know there
+aren't and conjecture why not. If so, describe them and provide links. How well 
+do they address the need? Are there any particularly admirable qualities of the
+language? Are there parts of the language you think could be improved?_
+There are no other DSLs which seems crazy! Someone should definitely change that. If it was easier to understand maybe some law makers would actually _read_ legislation instead of having their workers do it for them.
```

Comment:
... Maybe think of why this became the current system and why no other has taken over. It really seems like you wrote this just to soap box about not liking legalese. I feel like developing a DSL is about understanding why things are done certain ways. To make a better legal language, you should understand why legalese is widely accepted even though it can be hard to understand.